hello-world-attest-fn-call/as/log.go

```diff
@@ -0,0 +1,22 @@
+package as
+
+import (
+	"fmt"
+	"runtime"
+)
+
+// Imported host functions and supporting input/output types
```

Comment:
This comment is confusing.  It looks like it may have been related to some types that were removed?

Reply:
removing